config/crd/bases/logging.openshift.io_kibanas.yaml

```diff
@@ -161,7 +161,7 @@ spec:
             required:
             - managementState
             type: object
-          status:
+          statuss:
```

Comment:
This is the effect of the type in `kibana_types.go`. It will break CRD status subresource usage.

Reply:
Understood. Could you point me where the change could break existing usage? After installing the new manifest with the update CRD, the following operators succeed:
1. go test -run TestKibana
2. the kibana dashboard can display the logs from CLO
3. `oc get kibana -n openshift-logging -o yaml` 's output https://paste.centos.org/view/raw/ad164f17